spec/models/rent_spec.rb

```diff
@@ -0,0 +1,14 @@
+require 'rails_helper'
+
+describe Rent do
+  subject(:rent) { build(:rent) }
+
+  it { is_expected.to be_valid }
+
+  it { is_expected.to validate_presence_of :user }
+  it { is_expected.to validate_presence_of :book }
+  it { is_expected.to validate_presence_of :from }
+  it { is_expected.to validate_presence_of :to }
+
+  it { expect(rent.from).to be <= rent.to }
```

Comment:
I like the idea of this! What do you think about moving this to a validation in the rent? So we can avoid the creation of invalid rents when the `to` is less than the `from`